code-style/scss/README.md

```diff
@@ -1,52 +1,257 @@
-Naming Conventions
-=====
+# SCSS
 
-We follow [SMACSS](http://smacss.com/) conventions when naming CSS classes.
-When you are naming something, **do not** name it based on its content, as this will be difficult to
-use the module elsewhere. Try to name modules based on function instead.
+**Table of Contents**
 
-**Modules** are the building blocks of a website. Multi-word module names should be *hyphen-separated*:
-```css
-.module {}
-/* or */
-.even-grid {}
+1. [Introduction](#Introduction)
+2. [Class Naming Theory](#Class-Naming-Theory)
+3. [BEM](#BEM)
+  1. [Using BEM](#Using-BEM)
+    1. [Block](#Block)
+    2. [Element](#Element)
+    3. [Modifier](#Modifier)
+  2. [BEM in Sass](#BEM-in-Sass)
+4. [ITCSS](#ITCSS)
+  1. [File Naming](#File-Naming)
+    1. [Settings](#Settings)
+    2. [Tools](#Tools)
+    3. [Generic](#Generic)
+    4. [Elements](#Elements)
+    5. [Objects](#Objects)
+    6. [Components](#Components)
+    7. [Utilities](#Utilities)
+5. [Name Spacing](#Name-Spacing)
+6. [ITCSS and BEM](#ITCSS-and-BEM)
+
+
+## Introduction
+
+Our CSS is organized using [Harry Roberts](csswizardry.com)' Inverted Triangle CSS (ITCSS) organizational approach. This method is mixed with [Block Element Modifier](http://getbem.com) (BEM) naming convention for class names throughout the stylesheet.
+
+---
+
+## Class Naming Theory
+
+When it comes to naming classes, we find it best to be as specific as possible with the fewest words possible. This can be an exercise in understanding the purpose and function of a component. It's helpful to find like components and think about what they have in common and what their purpose is. You can then choose a generic name that encapsulates the uses the component will have.
+
+Keep in mind that naming is for humans, so verboseness is useful if it helps communicate what the class does. Ideally, a future developer should be able to look at a class name and have a reasonable idea of what they might see on the rendered page.
+
```

Comment:
I would love an example here! 

What is a good use case for using a full word in a class name? What is a good example of abbreviation?